Configuration/NodeTypes.Finishers.SaveFormData.yaml

```
@@ -1,10 +1,39 @@
 'PunktDe.Form.Persistence:SaveFormDataFinisher':
   superTypes:
-    'Neos.Form.Builder:AbstractFinisher': TRUE
+    'Neos.Form.Builder:AbstractFinisher': true
   ui:
     label: 'Save Form Data Finisher'
     icon: 'fas fa-save'
     inspector:
+      tabs:
+        default:
+          label: i18n
+          position: 10
+          icon: 'icon-pencil'
       groups:
-        'finisher':
-          icon: 'fas fa-save'
+        dataMail:
+          tab: default
+          label: i18n
+          icon: mail-bulk
```

Nitpick : I thought you were supposed to use the font awesome notation for icons 
```suggestion
          icon: 'fas fa-mail-bulk'
```